ui/src/app/utils/zendesk.ts

```diff
@@ -106,7 +106,9 @@ export function openZendeskWidget(
   // this property is dynamically loaded by the Zendesk web widget snippet,
   // and can't properly be typed. If for some reason the support widget is
   // unavailable, we'll show a notice to the user.
-  if (window['zE'] == null) {
+  // eslint-disable-next-line @typescript-eslint/dot-notation
+  const zE = window['zE'];
```

Comment:
The comment above indicates that this string-reference is desired, so let's make an exception